packages/mui-system/src/styleFunctionSx/defaultSxConfig.js

```diff
@@ -70,6 +70,10 @@ const defaultSxConfig = {
     themeKey: 'palette',
     transform: paletteTransform,
   },
+  outlineColor: {
+    themeKey: 'palette',
+    transform: paletteTransform,
```

Comment:
```suggestion
```
I think the `outlineColor` should be the same as `borderColor` ?

Reply:
You are correct. Done.